src/libs/actions/Report.js

```diff
@@ -1016,26 +1026,32 @@ function openReport(reportID) {
  * @param {Number} oldestActionSequenceNumber
  */
 function readOldestAction(reportID, oldestActionSequenceNumber) {
-    API.read('ReadOldestAction',
+    API.write('ReadOldestAction',
```

Comment:
NAB hahaha... we're `write`ing an action called `Read`? That doesn't make a whole lot of sense. The API command should probably be something more like `MarkOldestActionAsRead`

Reply:
If this is a NAB - I prefer the current name and think it's OK. The alternative suggestion confuses me a bit because I am imagining the user manually marking something (and this is something that happens as a side-effect of the user reading something with their eyes and not "reading data from the server").

Reply:
> (and this is something that happens as a side-effect of the user reading something with their eyes and not "reading data from the server").

+1 - Since the action is the "reading" I think it's fine, but maybe we could go with `ViewOldestAction` so we don't use the term "read"

Reply:
😅  I think this is still a read, and I just changed the wrong method, I meant to change OpenReport to Write